src/applications/check-in/locales/en/translation.json

```diff
@@ -83,6 +82,7 @@
   "no": "No",
   "not-available": "Not available",
   "online-check-in-isnt-available-for-this-appointment-check-in-with-a-staff-member": "Online check-in isn’t available for this appointment. Check in with a staff member.",
+  "or-you-can-login-to-your-va-account-to-update-your-contact-information-online": "Or you can <0>{{link}}</0> to your VA account to update your contact information online.",
```

Comment:
I think we can also separate the spanish translation without needing to re-send to the state department

Reply:
thanks for catching that, for some reason I was assuming that there wasn't a translation key for this in Spanish yet.